locales-pending/premium.ftl

```diff
@@ -187,16 +187,16 @@ dashboard-top-banner-lets-keep-protecting-title = Let’s keep protecting your d
 # $remaining_exposures_total_num is the remaining number of exposures the user has to resolve.
 dashboard-top-banner-lets-keep-protecting-description = 
   { $remaining_exposures_total_num -> 
-    [one] You still have { $remaining_exposures_total_num} exposure left to fix. Keep going and protect yourself. We’ll guide you step by step.
-    *[other] You still have { $remaining_exposures_total_num} exposures left to fix. Keep going and protect yourself. We’ll guide you step by step.
+    [one] You still have { $remaining_exposures_total_num } exposure left to fix. Keep going and protect yourself. We’ll guide you step by step.
+    *[other] You still have { $remaining_exposures_total_num } exposures left to fix. Keep going and protect yourself. We’ll guide you step by step.
   }
 dashboard-top-banner-lets-keep-protecting-cta = Let’s keep going
 
 dashboard-top-banner-your-data-is-protected-title = Your data is protected
 # Variables: 
-# $starting_exposures_total_num is the number of exposures the user has resolved.
+# $starting_exposure_total_num is the number of exposures the user has resolved.
 dashboard-top-banner-your-data-is-protected-description = 
-  { $starting_exposures_total_num ->
+  { $starting_exposure_total_num ->
```

Comment:
Looks like we're using the singular "exposure" form at:
https://github.com/mozilla/blurts-server/blob/ca76b4e647e421f5802b44064572051215cffee2/src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx#L98-L103

Was:

> File path: premium.ftl
> Message ID: dashboard-top-banner-your-data-is-protected-description
> Position: line 196 column 1
> Error (VC01): Messages including variables should have a comment explaining what will replace the variable. This can be either a message or group comment. 
> Missing references: `$starting_exposure_total_num`